source/Calamari.Shared/Deployment/Conventions/PackagedScriptConvention.cs

```diff
@@ -23,10 +23,10 @@ public void Install(RunningDeployment deployment)
                 return;
             }
 
-            RunScripts(deployment);
+            RunPreferredScript(deployment);
             if (deployment.Variables.GetFlag(SpecialVariables.DeleteScriptsOnCleanup, true))
             {
-                DeleteScripts(deployment);
+                DeletePreferredScript(deployment);
```

Comment:
We should delete all scripts, this matches the previous behaviour when both bash and ps scripts were included